docs/specifications/tests/Consistency-TP/consistency04.md

```diff
@@ -1,52 +1,84 @@
-## CONSISTENCY04: Name server NS consistency
+# CONSISTENCY04: Name server NS consistency
 
-### Test case identifier
+## Test case identifier
 
-**CONSISTENCY04:** Name server NS consistency
+**CONSISTENCY04**
 
-### Objective
+## Objective
 
 All authoritative name servers must serve the same NS record set
-(section 4.2.2) of [RFC 1034](https://tools.ietf.org/html/rfc1034)
-for the tested domain. Any inconsistency of NS records descibed in
-section 3.3.11 of [RFC 1035](https://tools.ietf.org/html/rfc1035)
-might result in operational failures.
+for the tested domain, child zone ([RFC 1034], section 4.2.2).
+Any inconsistency of NS records described in [RFC 1035], 
+section 3.3.11, might result in operational failures.
 
 The objective of this test is to verify that the NS records are
-consistent between all the authoritative name servers.
+consistent between all authoritative name servers of the child zone.
 
-### Inputs
+## Inputs
 
-1. The domain name to be tested
+The domain name to be tested ("child zone").
 
-### Ordered description of steps to be taken to execute the test case
+## Ordered description of steps to be taken to execute the test case
 
-1. Obtain the list of name servers from [Method2](../Methods.md) and
-   [Method3](../Methods.md).
-2. Retrieve the NS RR set from all the name servers. 
-3. If the NS RR set is not give the same answer from all the name
-   servers, this test case fails.
+1. Obtain the set of name server IPs for the *child zone* using
+   [Method4] and [Method5]. 
 
-### Outcome(s)
+2. Retrieve the NS RR set for the *child zone* from all retrieved 
+   name server IP address.
```

Comment:
The flow of this TC would be clearer if this step confined itself to querying the name server IPs and collecting the responses.